packages/client-sdk-nodejs/package.json

```diff
@@ -55,6 +54,7 @@
     "@gomomento/generated-types": "0.96.0",
     "@gomomento/sdk-core": "file:../core",
     "@grpc/grpc-js": "1.9.0",
+    "@types/google-protobuf": "3.15.10",
```

Comment:
I was getting some weird `@types/google-protobuf` is missing error when installing our sdk. So adding this back here. This probably means that we are leaking some grpc types somewhere, and this pr is not aiming at fixing that.